lib/stsci/tools/cfgpars.py

```diff
@@ -263,7 +263,8 @@ def getParsObjForPyPkg(pkgName, strict):
         flist = [f for f in flist if os.path.basename(f) == tname+'.cfg']
         if len(flist) > 0:
             theFile = flist[0]
-            assert len(flist) == 1, str(flist) # should never happen
+            if len(flist) != 1:  # should never happen
+                raise ValueError(str(flist))
```

Comment:
I can't imagine under what circumstances this could happen: two different files with the same file name in the same directory. Is it possible?

Reply:
No idea! I am just getting rid of assert. I try not to think too hard about the code, otherwise, I might refactor everything.

https://docs.openstack.org/bandit/latest/plugins/b101_assert_used.html